examples/OECC/config/rc.local

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+#
+
+export DLC=/psc/dlc
+export PATH=$DLC/bin:$PATH
+
+cd /psc/wrk
+proserve sports2020 -S 20000 -n 30
+
+cd /psc/wrk/oepas1
```

Comment:
Do we need to change directory if the script is fully pathed to the binary to run?

Reply:
Changing the directory is a practice to run scripts to ensure the current working directory independently on whether the full path to the binary is used.

Inspection of open files of PASOE suggests that it does not seem to be dependent on the current working directory.

Changed code to so that both proserve and tcman.sh use /psc/wrk/ as the working directory.